Frontend/ui-driver/src/Main.purs

```diff
@@ -50,9 +50,8 @@ main event = do
     _ <- runExceptT $ runBackT $ updateEventData event
     resp ← runExceptT $ runBackT $ Flow.baseAppFlow true
     case resp of
-      Right _ -> liftFlow $ main event
```

Comment:
Why are we removing this? The intention of adding it here was if in any case baseAppFlow returns and it was still required for user actions. App will not look freezed, we can keep this until it's causing any harm